### PR TITLE
Fix Replicant Delete

### PR DESCRIPTION
**Pull request by ismirlia**
When the pi_replicant field is set in terraform it creates a bunch of vms as expected, but terraform destroy does not delete these.

JIRA: https://jsw.ibm.com/browse/POWERIAAS-12494

```
--- PASS: TestAccIBMPIInstanceBasic (875.01s)
PASS
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_instance.go [hunk 1]
```diff
@@ -463,10 +467,9 @@ func resourceIBMPIInstanceRead(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(err)
 	}
 
-	cloudInstanceID, instanceID, err := splitID(d.Id())
-	if err != nil {
-		return diag.FromErr(err)
-	}
+	idArr := strings.Split(d.Id(), "/")
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_instance.go:470]: This does that
```suggestion
	idArr :=  flex.IdParts(d.Id())
```

Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_instance.go]: Made changes

**Review by michaelkad — APPROVED** [review]
Lgtm

**Comment by ismirlia** [comment]
Opened in main repo closing...

**Pull request closed without merging**